src/components/Actions/Actions.vue

```diff
@@ -662,6 +598,72 @@ export default {
 			this.$emit('blur', event)
 		},
 	},
+	/**
+	 * The render function to display the component
+	 *
+	 * @param {Function} h The function to create VNodes
+	 * @return {VNodes} The created VNodes
+	 */
+	render(h) {
+		// Get the actions
+		const actions = this.$slots.default || []
+
+		// Check that we have at least one action
+		if (actions.length === 0) {
+			return
+		}
+
+		/**
+		 * If we have a single action only and didn't force a menu,
+		 * we render the action as a standalone button
+		 */
+		if (this.isValidSingleAction && !this.forceMenu) {
+			console.debug('render single action')
+			return h('Button',
+				{
+					class: 'action-item action-item--single',
```

Comment:
```suggestion
					class: 'action-item action-item--single button-vue',
```

Haven't tested fully but seems this is needed to avoid leaking server styles

Reply:
Yes, there is still a lot of stuff missing and styles are not correct yet. I will fix these "details" once it's clear it works in principle and fixes the problems targeted. I clarified this in the first post.